### ISSUE TITLE
Could you improve the default new document ? with at least a section H1

### ISSUE THREAD

**Issue by bourgeoa** [dokieli/dokieli#271]
The missing of a section H1 makes it difficult to begin a new document there could be a default one

**Comment by csarven** [comment]
I agree. This is quite annoying. The bug originates in the MediumEditor (ME) library eg probably https://github.com/yabwe/medium-editor/issues/319 . We need a workaround for this.

The intended behaviour in ME was that typing at the location of the placeholder for h1 would work..

**Comment by csarven** [comment]
Resolved by https://github.com/dokieli/dokieli/pull/431 🎉
